components/payments/settings/payment-webhooks.tsx

```diff
@@ -7,5 +15,76 @@ interface PaymentsWebhooksProps {
 export const PaymentsWebhooks: React.FC<PaymentsWebhooksProps> = ({
   accountId,
 }) => {
-  return <Flex flexDir="column" gap={8}></Flex>;
+
+  const [productionWebhooks, setProductionWebhooks] = React.useState<PaymentsWebhooksType[]>([]);
+  const [testnetWebhooks, setTestnetWebhooks] = React.useState<PaymentsWebhooksType[]>([]);
+
+  const { data: webhooks, isLoading, isFetched, error } = usePaymentsWebhooksByAccountId(accountId);
+
+  React.useEffect(() => {
```

Comment:
never use useEffect and states for things that can be got from a different place, you can use a useMemo here which is much better

`const { productionWebhooks, testnetWebhooks } = useMemo(() => ... return { productionWebhooks, testnetWebooks}, [webhooks])`

lmk if that makes sense